fv3core/stencils/fv_dynamics.py

```diff
@@ -169,10 +169,13 @@ def do_dyn(state, comm):
         domain=grid.domain_shape_standard(),
     )
     print("DynCore", grid.rank)
+    timer.start("DynCore")
     dyn_core.compute(state, comm)
+    timer.stop("DynCore")
```

Comment:
Please use the context manager here and in the below blocks.
```suggestion
    with timer.clock("DynCore"):
        dyn_core.compute(state, comm)
```

Reply:
I'd also ask we use pothole case for these timing labels.